onnxruntime/core/optimizer/shape_to_initializer.cc

```diff
@@ -1,80 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#include "core/optimizer/shape_to_initializer.h"
-#include "core/graph/graph.h"
-#include "core/graph/graph_utils.h"
-#include "core/graph/op.h"
-#include "core/optimizer/initializer.h"
-#include "core/optimizer/optimizer_execution_frame.h"
-#include "core/framework/op_kernel.h"
-#include "core/framework/tensorprotoutils.h"
-
-namespace onnxruntime {
-
-Status ShapeToInitializer::Apply(Graph& graph, Node& node, RewriteRuleEffect& rule_effect, const logging::Logger&) const {
-  // Store the statically inferred shape of the input to the Shape operator.
-  const ONNX_NAMESPACE::TensorShapeProto* input_shape_proto = node.InputDefs()[0]->Shape();
```

Comment:
Retiring this RewriteRule as its functionality is already offered by another GraphTransformer (ConstantFolding) at the same optimization level (Level 1). Updating this would have been a copy/paste of the new logic added to the Shape processing logic in ConstantFolding